CHANGELOG.md

```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.13.0.dev0 (April 30th, 2020)
+
+The 0.13.0.dev0 is a *pre-release* version. To install it, use `pip install httpx --pre`.
+
+This release switches to `httpcore` for all the internal networking, which means:
+
+* We're using the same codebase for both our sync and async clients.
+* HTTP/2 support is now available with the sync client.
+* We no longer have a `urllib3` dependency for our sync client, although there is still an *optional* `URLLib3Dispatcher` class.
+
+It also means we've had to remove our UDS support, since maintaining that would have meant having to push back our work towards a 1.0 release, which isn't a trade-off we wanted to make.
```

Comment:
Should we include the rest of PRs that got in since 0.12.1? I believe there are a couple of them. We should also get them listed in the changelog.

Reply:
Gotcha. Done!